src/sections/privacidade/index.jsx

```diff
@@ -0,0 +1,243 @@
+import React from "react";
+import Typography from "@/components/typography";
+import Button from "@/components/button";
+import privacidadeEsquema from "@/assets/privacidade-esquema.png";
+import { useStaticQuery, graphql } from "gatsby";
+import SectionContent from "../section-content/index";
+
+import styles from "./index.module.css";
+
+const query = graphql`
+  query {
+    site {
+      siteMetadata {
+        appSiteUrl
+      }
+    }
+  }
+`;
+
+export default function Privacidade() {
+  const data = useStaticQuery(query);
+  return (
+    <div className={styles.root}>
+      <div className={styles.container}>
+        <SectionContent title="Segurança e Privacidade de Dados">
+          <Typography variant="smallBody">
+            Os dados de sintomas armazenados não são identificáveis e não são
+            correlacionáveis com dados de contacto. Isto foi possível graças à
+            forma especial como os dados são tratados e armazenados. Veja mais
+            em{" "}
+            <Typography
+              variant="smallBody"
+              weight="bold"
+              color="purple"
+              className={styles.inlineText}
+            >
+              Como tratamos os seus dados.
```

Comment:
This should be an anchor to "Como tratamos os seus dados" section, right?

Reply:
Done!